prometheus/ready.go

```diff
@@ -128,22 +180,30 @@ func WaitForReady(inCtx context.Context, inCtxCancel context.CancelFunc, cfg con
 			respOK := err == nil && resp.StatusCode/100 == 2
 
 			if respOK {
-				result, err := scrapeMetrics(inCtx, cfg)
```

Comment:
Note: This was a bug. `err` is in a new scope so the outer block prints `err=null` in the log statement, means we never printed "waiting for scrape interval" messages in prior releases.